Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/ui/consent/FamilyTestConsentViewModel.kt

```diff
@@ -29,7 +28,7 @@ class FamilyTestConsentViewModel @AssistedInject constructor(
 
     val isSubmittable = combine(personName, registrationStateProcessor.state) { name, state ->
         name.isNotEmpty() && state !is TestRegistrationStateProcessor.State.Working
-    }.asLiveData()
+    }.asLiveData2()
```

Comment:
@schauersbergern  FYI, this was the fix `asLiveData` was not using the dispatcher provider that you pass in the test 
either use pass it `asLiveData(dispatcher.Default)` or use `asLiveData2` which does it for you ;)

Reply:
@mtwalli Perfect, thanks for checking!